openfisca_tracker/tests/test_piwik.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+from nose.tools import assert_equal
+
+from openfisca_tracker.piwik import PiwikTracker
+
+TRACKER_URL = 'https://openfisca.innocraft.cloud/piwik.php'
+TRACKER_IDSITE = 1
+
+tracker = PiwikTracker(TRACKER_URL, TRACKER_IDSITE)
+
+
+# Doesn't check that the action was actually tracked, just that it doesn't crash.
+# You can manually check online at TRACKER_URL that the action was indeed tracked.
+def test_track():
+    tracker.track('https://api-test.openfisca.fr/variables')  # thread
```

Comment:
Removed. Its purpose was to indicate that an instance with a specific life cycle existed here and that it might be tested.